.circleci/config.yml

```diff
@@ -10,30 +10,43 @@ jobs:
       tag: 16.13-browsers
     steps:
       - checkout
+      - run:
+          name: Update npm
+          command: 'sudo npm install -g npm@latest'
       - restore_cache:
-          key: node_modules-{{ checksum "package-lock.json" }}-{{ checksum "package.json" }}
+          key: dependency-cache-{{ checksum "package-lock.json" }}
       - run:
           name: Install Dependencies
           command: CYPRESS_INSTALL_BINARY=0 npm ci --no-audit
       - save_cache:
-          key: node_modules-{{ checksum "package-lock.json" }}-{{ checksum "package.json" }}
+          key: dependency-cache-{{ checksum "package-lock.json" }}
           paths:
-            - ./node_modules
+            - node_modules
+            - ~/.cache
+      - run: NODE_ENV=production npm run build
       - run:
           name: Run eslint
           command: npm run lint
-      - run:
-          name: Run Tests
-          command: |
-             set -e
-             npm test -- --maxWorkers=2
-      - run: NODE_ENV=production npm run build
       - persist_to_workspace:
           root: .
           paths:
             - node_modules
             - build
 
+  unit_test:
+    executor:
+      name: hmpps/node
+      tag: 16.13-browsers
```

Comment:
Could possibly extract into a [param](https://github.com/ministryofjustice/hmpps-template-typescript/blob/main/.circleci/config.yml#L19) to allow reuse but no big deal.